can/interfaces/socketcand/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""
+Interface to socketcand
+see https://github.com/linux-can/socketcand
+
+Copyright (C) 2021  DOMOLOGIC GmbH
+http://www.domologic.de
```

Comment:
A HTTPS link would be nice, can it be that your website does not support that? 🙃

Reply:
Yes, HTTPS is currently not supported - the website is quite old, it is only a kind of virtual name card . We will support it HTTPS by beginning of next year. Is the http link a problem for you?

Reply:
We can leave it there, I just wanted to give you a "heads up" on this.